commands/acl.md

```
@@ -0,0 +1,135 @@
+
+Usage: `ACL GETUSER <username>`
+
+#### Example
+
+```
+> ACL GETUSER john
+```
+
+#### Output
+
+```
```

Markdownlint (0.35.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)